.github/workflows/integrate.yml

```diff
@@ -30,6 +30,34 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: TODO
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install robotpy
+        run: |
+          python -m pip install --upgrade pip
+          python3 -m pip install robotpy
+      - name: Sync robotpy
+        run: |
+          python3 -m robotpy sync
+      - name: Install dependencies
+        run: |
+          python3 -m pip install ruff
+          python3 -m pip install coverage
+          python3 -m pip install opencv-python
+      - name: Run tests
+        run: |
+          python3 -m robotpy test
+      - name: Generate coverage report
+        run: |
+          python3 -m robotpy coverage test
+          coverage html -d coverage_html
+      - name: HTML Preview
+        id: html_preview
+        uses: pavi2410/html-preview-action@v2
+        with:
+          html_file: "index.html"
```

Comment:
This [action](https://github.com/marketplace/actions/html-preview) currently isn't working.

There may be another way to host the generated coverage html file on github.